packages/MJCore/src/generic/entityInfo.ts

```diff
@@ -985,11 +968,11 @@ export class RecordMergeRequest {
     /**
      * The primary key value(s) for the surviving record - if the entity in question has a single-valued primary key this will be an array with a single element, otherwise it will be an array with multiple elements
      */
-    SurvivingRecordPrimaryKeyValues: PrimaryKeyValue[]
+    SurvivingRecordKeyValuePairs: KeyValuePair[]
```

Comment:
@JS-BC  per earlier comments, this is a shortcut and you will LOSE information here if CompositeKey class one day has other stuff beyond just the array of KeyValuePairs, so clean this up and use the CK class pls